src/LiveQuery/ParseLiveQueryServer.js

```diff
@@ -901,8 +899,15 @@ class ParseLiveQueryServer {
         subscription: subscription,
       };
       // Add selected fields, sessionToken and installationId for this subscription if necessary
+      if (request.query.keys) {
+        subscriptionInfo.keys = Array.isArray(request.query.keys) ? request.query.keys : [request.query.keys];
```

Comment:
This appears to be the right way to do this IMO. I previously had to fix some issues in #7242 where the server wan't properly digesting JSON arrays and instead relied on comma separated strings